File: python/Node.py
class Node:
    def __init__(self, data, left_child, right_child):
        self.data = data
        self.left_child = left_child
        self.right_child = right_child

    @property
    def data(self):
        return self.__data

    @data.setter
    def data(self, data):
        self.__data = data

    @property
    def left_child(self):
        return self.__left_child

    @left_child.setter
    def left_child(self, left_child):
        self.__left_child = left_child

    @property
    def right_child(self):
        return self.__right_child

    @right_child.setter
    def right_child(self, right_child):
        self.__right_child = right_child

    def __repr__(self):
        return "{" + "data:{data}, leftChild:{leftChild}, rightChild:{rightChild}".format(data=self.data,
                                                                                          leftChild=self.left_child,
                                                                                          rightChild=self.right_child) + "}"

File: php/Node.php
<?php

class Node
{
    private $data;
    private $leftChild;
    private $rightChild;

    public function __construct($data, $leftChild, $rightChild)
    {
        $this->data = $data;
        $this->leftChild = $leftChild;
        $this->rightChild = $rightChild;
    }

    /**
     * @return string
     */
    public function getData()
    {
        return $this->data;
    }

    /**
     * @return string
     */
    public function getLeftChild()
    {
        return $this->leftChild;
    }

    /**
     * @return string
     */
    public function getRightChild()
    {
        return $this->rightChild;
    }

    /**
     * @param string $data
     */
    public function setData($data)
    {
        $this->data = $data;
    }

    /**
     * @param string $leftChild
     */
    public function setLeftChild($leftChild)
    {
        $this->leftChild = $leftChild;
    }

    /**
     * @param string $rightChild
     */
    public function setRightChild($rightChild)
    {
        $this->rightChild = $rightChild;
    }
}